docs/user/extract-text.md

```diff
@@ -20,6 +20,21 @@ print(page.extract_text(0))
 print(page.extract_text((0, 90)))
 ```
 
+you can also extract text in "layout" mode:
```

Comment:
```suggestion
You can also extract text in "layout" mode:
```

Reply:
Should probably be changed in the lines above as well, where I cannot add a suggestion as it is not part of your changes.